account_analytic_tag/views/account_analytic_line_views.xml

```diff
@@ -0,0 +1,24 @@
+<odoo>
+    <!-- Inherit Form View to Modify it -->
+    <record id="view_account_analytic_line_tree_inherit_tags" model="ir.ui.view">
+        <field name="name">account.analytic.line.tree.inherit.tags</field>
+        <field name="model">account.analytic.line</field>
+        <field name="inherit_id" ref="analytic.view_account_analytic_line_tree" />
+        <field name="arch" type="xml">
+            <field name="account_id" position="after">
+                <field name="tag_ids" widget="many2many_tags" optional="hide" />
```

Comment:
Can you make this field visible only to the `account_analytic_tag.group_analytic_tags` group?